packages/bundle/package.json

```
@@ -29,7 +29,7 @@
     "start:webpack": "npm run build:webpack -- --watch"
   },
   "dependencies": {
-    "@babel/runtime": "7.11.0",
+    "@babel/runtime": "^7.11.2",
```

For production dependencies, please use `npm install --save-exact` or `lerna --exact` to lockdown on versions.

It is okay to hand modify this by removing the caret, it will not affect `package-lock.json`.

```suggestion
    "@babel/runtime": "7.11.2",
```